### ISSUE TITLE
better feedback for comments with an open changeset

### ISSUE THREAD

**Issue by HolgerJeromin** [openstreetmap/openstreetmap-website#845]
Accidentally i uploaded a lot of temp data. I corrected all of them and had luck, the changeset was still open so i could update the description.
But it would be nice if i could comment a changeset, in case someone come across this in the future.

**Comment by ukasiu** [comment]
I just thought that it's useless to comment on unfinished changeset, because of their volatility, but well, I think this probably was an oversight.

**Comment by jgpacker** [comment]
Perhaps the issue here is just that we need to make it clearer the changeset cannot be discussed exactly because it's still open.

**Comment by matkoniecz** [comment]
> Perhaps the issue here is just that we need to make it clearer the changeset cannot be discussed exactly because it's still open.

Yes, I encountered this and I was confused by missing comment box (I needed to ask on IRC to discover what is happening).

**Comment by mmd-osm** [comment]
> Yes, I encountered this and I was confused by missing comment box (I needed to ask on IRC to discover what is happening).

Pull request #1705 adds some further explanations now as to why an open changeset cannot be commented yet.
